custom_components/owlintuition/sensor.py

```diff
@@ -288,7 +288,7 @@ def __init__(self, owldata, sensor_name, sensor_type, phase=0, zone=1, zones_cou
             self._name += f' P{phase}'
         self._zone = zone
         self._name_zone_updated = (zones_count == 1)
-        self._state = None
+        self._attr_native_value = None  # `_state` is now a cached property by HA, so we use this instead (see #36
```

Comment:
I think `_attr_native_value` is set to None by default

Reply:
will drop the line then, and add a (still useful) comment in the `update` method